src/Spryker/Zed/EventBehavior/Business/Model/TriggerManager.php

```diff
@@ -76,7 +74,13 @@ public function triggerRuntimeEvents()
         }
 
         $processId = RequestIdentifier::getRequestId();
-        $events = $this->queryContainer->queryEntityChange($processId)->find()->getData();
+        try {
+            $events = $this->queryContainer->queryEntityChange($processId)->find()->getData();
+        } catch (PropelException | ConnectionException $e) {
```

Comment:
Do we need to catch both here? Is there not only one exception you can catch? Maybe it would make sense to add a check for a specific exception message?

Reply:
Yes, we need them both.
It's possible that we don't have any DB when we execute install. E.g. on the brand new install or if there was an error on DB creating. In that case on each install command application will throw that exception when it will try to fetch event records from DB.